client/src/ActionsAndReducers/dbWargames/wargames_ActionCreators.ts

```diff
@@ -306,12 +306,12 @@ export const updateForces = (dbName: string, newData: ForceData[]) => {
 }
 
 export const saveForce = (dbName: string, newData: ForceData) => {
-  return async (dispatch: WargameDispatch, state: any) => {
-    const oldForceData = state().wargame.data.forces.selectedForce
-    if (newData.iconURL !== oldForceData.iconURL && newData.iconURL !== forceTemplate.iconURL) {
-      const savedIconURL = await wargamesApi.saveIcon(newData.iconURL)
-      newData.iconURL = savedIconURL.path
-    }
+  return async (dispatch: WargameDispatch) => {

```

Comment:
@lilitkarapetyan - this next section is commented out.  We need to decide what to do with it, please.